docs/developer/deploy/thirdweb/one-click-deploy.md

```diff
@@ -0,0 +1,103 @@
+---
+title: One-Click Deploy
+---
+
```

Comment:
This file is now double

Reply:
I deleted the extra file. There were 2 tutorials, so I merged them into one but forgot to delete the old one